pandas/core/window.py

```diff
@@ -623,20 +624,30 @@ def _validate_win_type(win_type, kwargs):
                 arg_map = {'kaiser': ['beta'],
                            'gaussian': ['std'],
                            'general_gaussian': ['power', 'width'],
-                           'slepian': ['width']}
+                           'slepian': ['width'],
+                           'exponential': ['center', 'tau']}
+                immutable_args_map = {'exponential': {'center': None}}
+
                 if win_type in arg_map:
```

Comment:
why are you changing the structure of passing args here? was there some reason the existing would not just work?

Reply:
For exponential window kwargs contains ```{tau: 10}``` but I need to get back tuple ```(None, 10)```  (first ```None``` is the center parameter required for exponential window). So I pass the ```immutable_args``` to ```_pop_args``` function that is the second source of params (another than kwargs).

Another possible option can be add {"center": None} to kwargs for exponential window and let ```_pop_args``` without change.
More info about exponential window is in comments above.

Reply:
yes pls do this instead

Reply:
The code is now without `immutable_args`. As I wrote in another comment, I simplyfied my code even more because other window (that would be possibly implemented in future) in scipy will not need such handling like the exponential one. So please check the last code version. 
Thanks.